StripeCheckoutServerGatewayFactory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Payum\Stripe;
+
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Exception\LogicException;
+use Payum\Core\GatewayFactory;
+use Payum\Stripe\Action\CheckoutServer\CaptureAction;
+use Payum\Stripe\Action\CheckoutServer\ConvertPaymentAction;
+use Payum\Stripe\Action\CheckoutServer\StatusAction;
+use Payum\Stripe\Action\CheckoutServer\SyncAction;
+use Payum\Stripe\Action\Api\CreateSessionAction;
+use Payum\Stripe\Action\Api\RetrievePaymentIntentAction;
+use Stripe\Stripe;
+
+class StripeCheckoutServerGatewayFactory extends GatewayFactory
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function populateConfig(ArrayObject $config)
+    {
+        if (false === class_exists(Stripe::class)) {
```

Comment:
It's better to check for `PaymentIntent::class` because this class has been added on 6.9 thus we force the minimal version.
`Stripe::class` is about to check if sdk is installed

Reply:
Stripe has `const VERSION = '6.40.0';` so ultimately we should check against it. (Or at least I hope it appeared in earlier versions than 6.40).